### DISCUSSION TITLE
Default (crowd-sourced?) images per app/game

### DISCUSSION THREAD

**Discussion by Ariane-B** [LizardByte/community#108] [Sunshine]
I wasn't a big fan of the default images Sunshine was giving me for GameStream Launchpad for Playnite, so I quickly created some in GIMP:

![gs-playnite-4k](https://user-images.githubusercontent.com/6244273/208256823-06dd0062-2948-4c84-a976-58cf1b8e64dc.png)
![gs-playnite-1080p](https://user-images.githubusercontent.com/6244273/208256824-51159cc6-a5be-4882-ab65-d79838e91b1c.png)

It occurred to me that it might be cool if Sunshine had images ready for various popular apps. People could check something in their Sunshine config, and it'd allow Sunshine to upload their images to some sort of image repository, where it would be associated with tags that would help identify the program (metadata, `.exe` file name, etc.)

Then, if someone adds the same program to Sunshine but does not specify a custom image, Sunshine would fetch that default image and serve it to Moonlight.

**Comment by ReenigneArcher** [comment]
These look nice! I would suggest removing the rounded corners.

Perhaps an alternate idea is to expand the cover finder to search SteamGridDB (https://www.steamgriddb.com/game/5248053/grids)... although they use 2:3 aspect ratio.

**Reply by Ariane-B** [reply]
Here you go. I'll even throw in the GIMP file if you need it for anything.

[gs-playnite.zip](https://github.com/LizardByte/Sunshine/files/10252491/gs-playnite.zip)
![gs-playnite-4k](https://user-images.githubusercontent.com/6244273/208257418-73eda0c8-d2e3-4f00-8945-8731448efe7f.png)
![gs-playnite-1080p](https://user-images.githubusercontent.com/6244273/208257419-d212877d-55d4-464f-a65b-25eb3dc0a7f5.png)

As for Steam Grid... Sounds like a great idea. The aspect ratio sounds like it's probably close enough that it'd be possible to crop them into Moonlight's format and it'd mostly be fine. Or, alternatively, maybe you could talk to the people at Moonlight and convince them to support 2:3 aspect ratios too. Being compatible with Steam's big library of images would probably be a pretty big argument.
